presto-native-execution/pom.xml

```diff
@@ -204,6 +205,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
You can directly add this dependency in the profile xml itself

```
 <profile>
            <id>spark3</id>

            <activation>
                <property>
                    <name>upgrade</name>
                    <value>true</value>
                </property>
            </activation>

            <properties>
                <dep.pos.classloader.version>3</dep.pos.classloader.version>
            </properties>
            <dependency>
                <groupId>com.facebook.presto</groupId>
                <artifactId>presto-spark-classloader-spark${dep.pos.classloader.version}</artifactId>
                <scope>test</scope>
            </dependency>
        </profile>
```
You can add the `2` version in default profile